userguide/storage.rst

```diff
@@ -770,6 +781,47 @@ usually not worth the performance hit.
    dataset are saved, giving the effect of deduplication without the
    overhead.
 
+#ifdef freenas
+.. _Deduplication RAM requirements:
+
+Deduplication RAM requirements
+------------------------------
+
+A generally quoted rule of thumb is 5 GB of RAM per terabyte of 
+deduplicated storage. However, if the pool is highly duplicated 
+(around 3x to 5x space saving) this can drop to as low as 1 GB to 2 
+GB of RAM per TB of deduplicated storage. The exact size required for 
+deduplication data is found by using the :command:`CLI` command 
+:command:`zdb -U /data/zfs/zpool.cache -D pool_name` to identify the 
+total number of blocks in the pool, and the number of bytes of RAM 
+required per block (shown as *"{number} in core"* in the output). The 
+command  :command:`zdb -U /data/zfs/zpool.cache -S pool_name` 
+displays an estimate of the storage saving if deduplication is 
+applied to a dataset. Note that these commands may take a long time 
```

Comment:
Try to avoid using the word "note" to introduce a sentence, as the existence of the sentence means the reader is supposed to note it anyway.

Reply:
updated, please re-review